pyof/foundation/basic_types.py

```diff
@@ -114,13 +117,15 @@ class DPID(GenericType):
 
     _fmt = "!8B"
 
-    def __init__(self, dpid=None):
+    def __init__(self, value=None):
```

Comment:
`dpid` is a better variable name than `value` because it has more meaning. The same applies for similar modifications.